pages/submission/create.tsx

```diff
@@ -12,18 +41,39 @@ const CreateSubmissionPage = () => {
       <div className="relative mt-6 mr-10 ml-10 overflow-x-auto">
         <Header title={"New Submission"} />
         <hr className="mb-6" />
-        <div className="m-2">
-          <TextInput inputName="title" />
-          <TextAreaInput inputName="symptoms" />
-          <Button
-            className="mt-8 px-5 py-2 font-normal"
-            type="submit"
-            size="large"
-            onClick={() => {}}
-          >
-            Send submission
-          </Button>
-        </div>
+        <form
+          onSubmit={(e) => {
+            e.preventDefault();
+            handleSubmit((values) => {
+              mutateCreate.mutate(values);
+            });
+          }}
+        >
+          <div className="m-2">
+            <TextInput
+              inputName={"title"}
+              labelName={"Title"}
+              errors={errors}
+              {...register("title")}
+            />
+            <TextAreaInput
+              inputName={"symptoms"}
+              labelName={"Symptoms"}
+              errors={errors}
+              {...register("symptoms")}
+            />
+            <Button
+              className="mt-8 px-5 py-2 font-normal"
+              type="submit"
+              size="large"
+              onClick={handleSubmit((values) => {
+                mutateCreate.mutate(values);
```

Comment:
you don't need this here if it's of type submit,. it will call the form's submit handler automatically